pkg/commands/git.go

```diff
@@ -63,7 +63,7 @@ func NewGitCommand(
 		return nil, err
 	}
 
-	repo, err := setupRepository(gogit.PlainOpen, cmn.Tr.GitconfigParseErr)
+	repo, err := setupRepository(gogit.PlainOpenWithOptions, &gogit.PlainOpenOptions{DetectDotGit: false, EnableDotGitCommonDir: true}, cmn.Tr.GitconfigParseErr)
```

Comment:
Do we need to pass around a pointer here? If the struct isn't too large (e.g. just contains a small number of fields) I reckon passing the actual value would be easier

Reply:
We definitely do not need it.

Reply:
_But_ my reasoning was that [PlainOpenWithOptions](https://github.com/mark2185/lazygit/blob/803d9cc7d0584d0107bd085edfe3a1b89ea9c5b9/vendor/github.com/jesseduffield/go-git/v5/repository.go#L246) expects a pointer so if we're passing it as-is, this also needed to have the same signature?

Reply:
On a second look, `setupRepository` doesn't need the pointer, I'll change that.

Reply:
ah okay, if that's the case we can stick with the pointer approach

Reply:
Welll our hands are tied by the compiler, aren't they?